<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DistanceViewDialog</class>
 <widget class="QDialog" name="DistanceViewDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>252</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View distance</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>391</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Correct value</string>
   </property>
   <widget class="QLabel" name="ohorseYicongLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>-1 Yicong</string>
    </property>
   </widget>
   <widget class="QLabel" name="dhorseYicongLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>+1 Yicong</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ohorseLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="mashuLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ohorseYicongLineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dhorseLineEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="feiyingLineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="dhorseYicongLineEdit">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="offensiveHorseLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>-1 Horse</string>
    </property>
   </widget>
   <widget class="QLabel" name="mashuLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mashu</string>
    </property>
   </widget>
   <widget class="QLabel" name="feiyingLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>50</y>
      <width>42</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Feiying</string>
    </property>
   </widget>
   <widget class="QLabel" name="defensiveHorseLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>+1 Horse</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="leftDistanceLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>78</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Left distance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="rightDistanceLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="leftDistanceLineEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="rightDistanceLabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>84</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Right distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="finalResultLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>72</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Final result</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="finalResultLineEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="seatLabel">
      <property name="text">
       <string>Seat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="fromSeatLineEdit">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="toSeatLineEdit">
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="attackRangeLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Attack Range of Source</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="attackRangeLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="inAttackRangeLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>220</y>
     <width>90</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>In Attack Range</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inAttackRangeLineEdit">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="fromCombobox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="fromLabel">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>21</y>
     <width>24</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>From</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>to</string>
   </property>
  </widget>
  <widget class="QComboBox" name="toCombobox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="minorValueLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>66</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Minor value</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="minorValueLineEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
